arch/risc-v/src/qemu-rv/qemu_rv_timerisr.c

```diff
@@ -61,11 +61,13 @@
 
 void up_timer_initialize(void)
 {
+#ifndef CONFIG_BUILD_KERNEL
```

Comment:
I think this is unnecessary. The mtimer driver forwards access to SBI, if CONFIG_ARCH_USE_S_MODE is defined.

But this is fine also if you don't need the ticker.

Reply:
The SBI is not integrated yet, so the mtimer is disabled.